analysis_configs/s_channel_leptons_jetHT_data_pre_selection.py

```diff
@@ -90,7 +90,7 @@ def process(events, cut_flow, year, primary_dataset="", pn_tagger=False, **kwarg
     skimmer_utils.update_cut_flow(cut_flow, "METFilters", events)
 
     # Phi spike filter
-    events = skimmer_utils.apply_phi_spike_filter(events, year)
+    events = skimmer_utils.apply_phi_spike_filter(events, year, "skimmer/schannel_hot_spots.pkl", "s")
```

Comment:
Small question: is the data in the .pkl file doing exactly the same and the code? If you say so, we should at least tell Cesare to check it results in the same selection (or I can check that on one of his files).

Reply:
Yes, it is the same selection, doing the exact same thing as before for s-channel.